README.md

```diff
@@ -2,6 +2,8 @@
 
 Documentation is available [here](https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/).
 
+You can download a copy of the Software Bill of Materials (SBOM) [here](https://github.com/keptn/keptn/actions/runs/1416203195)
```

Comment:
Thank you for this PR @DavidPHirsch. :rocket: Also here, I'd see the merge of #471 as a precondition before merging this one - because only if this is done we will have the correct link to the resulting workflow.